src/src/CMakeLists.txt

```diff
@@ -134,22 +134,17 @@ endif()
 
 
 #---[ Demos ]------------------------------------------------------------
-# if (TILEDB_VS_ENABLE_BLAS)
-  # requires BLAS
-  add_executable(flat_l2 flat_l2.cc)
-  target_link_libraries(flat_l2 PUBLIC kmeans_lib nlohmann_json::nlohmann_json)
-  target_compile_definitions(flat_l2 PUBLIC TILEDB_VS_ENABLE_STATS)
-
-  add_executable(ivf_flat ivf_flat.cc)
-  target_link_libraries(ivf_flat PUBLIC kmeans_lib)
-  target_compile_definitions(ivf_flat PUBLIC TILEDB_VS_ENABLE_STATS)
-
-  add_executable(index index.cc)
-  target_link_libraries(index PUBLIC kmeans_lib)
-  target_compile_definitions(index PUBLIC TILEDB_VS_ENABLE_STATS)
-#else()
-#  message(STATUS "Disabling demo executables build because BLAS is disabled (TILEDBVS_ENABLE_BLAS).")
-#endif()
+if (TILEDB_VS_ENABLE_BLAS OR TRUE)
+  # these don't really requires BLAS any longer
```

Comment:
(Optional) Should we just remove the `TILEDB_VS_ENABLE_BLAS` and the if/else condition?